calypso/api.go

```diff
@@ -136,10 +140,7 @@ func (c *Client) Authorize(who *network.ServerIdentity, what skipchain.SkipBlock
 func (c *Client) DecryptKey(dkr *DecryptKey) (reply *DecryptKeyReply, err error) {
 	reply = &DecryptKeyReply{}
 	err = c.c.SendProtobuf(c.bcClient.Roster.List[0], dkr, reply)
-	if err != nil {
-		return nil, err
-	}
-	return reply, nil
+	return reply, cothority.ErrorOrNil(err, "sending DecryptKey message: %v")
```

Comment:
the function takes care of the `%v` part

Reply:
Removed the formatting verb